app.py

```diff
@@ -4,7 +4,7 @@
 import json
 
 from flask import Flask, request, make_response
-from daily_prayer import PrayerInfo
+from prayer_info import PrayerInfo
```

Comment:
Our prayer_info object below is named the same thing as our package - can we rename the variable below?